packages/gatsby-image/src/index.js

```diff
@@ -197,26 +211,35 @@ class Image extends React.Component {
       className,
       style = {},
       imgStyle = {},
+      objectFit,
+      objectPosition,
       placeholderStyle = {},
       fluid,
       fixed,
       backgroundColor,
       Tag,
     } = convertProps(this.props)
 
-    const bgColor = typeof backgroundColor === `boolean` ? `lightgray` : backgroundColor
+    const bgColor =
+      typeof backgroundColor === `boolean` ? `lightgray` : backgroundColor
 
     const imagePlaceholderStyle = {
       opacity: this.state.imgLoaded ? 0 : 1,
       transition: `opacity 0.5s`,
       transitionDelay: this.state.imgLoaded ? `0.5s` : `0.25s`,
+      objectFit: objectFit,
+      objectPosition: objectPosition,
+      fontFamily: `"object-fit: ${objectFit}; object-position: ${objectPosition}"`,
       ...imgStyle,
       ...placeholderStyle,
     }
 
     const imageStyle = {
       opacity: this.state.imgLoaded || this.state.fadeIn === false ? 1 : 0,
       transition: this.state.fadeIn === true ? `opacity 0.5s` : `none`,
+      objectFit: objectFit,
+      objectPosition: objectPosition,
+      fontFamily: `"object-fit: ${objectFit}; object-position: ${objectPosition}"`,
```

Comment:
because this is duplicated for both `imagePlaceholderStyle` and `imageStyle`, we should just add them to `imgStyle` object:

`imgStyle.objectFit = objectFit` before creating `imagePlaceholderStyle` object

Reply:
Nice! Will do.